root/edit/details/AddReleaseGroup.js

```diff
@@ -15,18 +15,21 @@ import ExpandedArtistCredit from
   '../../static/scripts/common/components/ExpandedArtistCredit';
 
 type AddReleaseGroupProps = {
```

Comment:
I'd prefer renaming this to
```suggestion
type AddReleaseGroupEditT = {
```
and adding a separate type for the React props like
```JS
type Props = {
  +allowNew?: boolean,
  +edit: AddReleaseGroupEditT,
};
```
because we'll likely be using these edit type types soon